src/Lykke.Job.BlockchainCashoutProcessor.Core/Domain/CrossClient/CrossClientCashoutState.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace Lykke.Job.BlockchainCashoutProcessor.Core.Domain
+{
+    public enum CrossClientCashoutState
+    {
+        StartedCrossClient,
```

Comment:
Since enum name already contains "CorssClient", I think, Started will be enough to name this item.